ceph.spec.in

```diff
@@ -266,6 +261,10 @@ BuildRequires:	python3-devel
 BuildRequires:	python3-setuptools
 BuildRequires:	python3-Cython
 %endif
+BuildRequires:	python%{_python_buildid}-prettytable
```

Comment:
i thought you were removing `python%{_python_buildid}-prettytable` from the `%if 0%{?suse_version}` block. but it seems you are not. is this intentional ?

Reply:
Actually what's happening here is I'm moving the entire conditional block

```
%if 0%{?rhel}
BuildRequires:	python34-devel
BuildRequires:	python34-setuptools
BuildRequires:	python34-Cython
%else
BuildRequires:	python3-devel
BuildRequires:	python3-setuptools
BuildRequires:	python3-Cython
%endif
```

into the `%if 0%{?fedora} || 0%{?rhel}` block. The git diff is confusing.

This commit is not changing the `python%{_python_buildid}-prettytable` (RHEL/Fedora)/`python%{_python_buildid}-PrettyTable` (SUSE) build dep at all.